syntax = "proto3";

package xain.protobuf.coordinator;

service Coordinator {
  rpc Rendezvous(RendezvousRequest) returns (RendezvousReply) {}
  rpc Heartbeat(HeartbeatRequest) returns (HeartbeatReply) {}
}

enum RendezvousResponse {
  ACCEPT = 0;
  LATER = 1;
}

message RendezvousRequest {}

message RendezvousReply {
  RendezvousResponse response = 1;
}

message HeartbeatRequest {}

message HeartbeatReply {}
